boards/snps/nsim_riscv/nsim_riscv_rmx500.yaml

```diff
@@ -0,0 +1,15 @@
+identifier: nsim_riscv/rmx500
+name: Synopsys rmx500
+simulation: nsim
+simulation_exec: nsimdrv
+type: sim
+arch: riscv
+toolchain:
+  - zephyr
+  - cross-compile
+  - arcmwdt
```

Comment:
I guess we don't support mwdt right now?

Reply:
Fixed